src/components/inline_edit/inline_edit_form.tsx

```diff
@@ -117,6 +127,7 @@ export const EuiInlineEditForm: FunctionComponent<EuiInlineEditFormProps> = ({
   editModeProps,
   isLoading,
```

Comment:
I mentioned this as being a bug in our last pairing session, but after looking a little closer at the EuiSkeleton components, I actually think that those defaulting to assuming the loading state is true makes sense. In which case, I think we should update EuiInlineEdit's props accordingly instead:

```suggestion
  isLoading = false,
```

BTW, by setting this at the `EuiInlineEditForm` level we can also remove the `= false` default for `EuiInlineEditText` and `EuiInlineEditTitle`.